src/CookieJar.php

```diff
@@ -310,7 +310,7 @@ public function merge($arg)
 
 		}, func_get_args());
 
-		$data = call_user_func_array('Arr::merge', $arguments);
+		$data = call_user_func_array(__NAMESPACE__.'\Arr::merge', $arguments);
```

Comment:
I think we should go for @WanWizard's idea of having an instance of `Arr` created in the constructor then we can make it injectable via that with the DIC later on.